tests/config_test.go

```diff
@@ -68,6 +71,93 @@ var _ = Describe("[Serial][rfe_id:899][crit:medium][vendor:cnv-qe@redhat.com][le
 		}
 	}
 
+	withSecret := func(secretName string, volumeName string, customLabel ...string) libvmi.Option {
```

Comment:
nit of my own: named lambdas are useful when you change their value to another function, or when the function uses a variable from its scope of definition. On other occasions, the simple `func withSecret` syntax is more readable and less "frightening" - as a reader I don't need to worry that the function may depend on a context or change itself. It also allows to write it down at the bottom of the file, so a reader can see the test, understand it, and not be troubled by the implementation detail.

Reply:
@dankenigsberg Thank you, makes sense to me as well. PTAL at the changes

Reply:
thanks! I think it makes the file more readable.

As a reviewer, I find it confusing to see you add the functions here, only to be moved to file context in another commit. I think that `e2e: utils.go: remove redundant functions` should be limited to removing old functions, and not including moving of these new ones.